profile.md

```diff
@@ -130,6 +130,7 @@ The following table specifies which properties MUST appear in each API:
 | --- | --- |
 | `PUT /v2/service_instances/:instance_id` | `organization_guid`<br>`space_guid` |
 | `PATCH /v2/service_instances/:instance_id` | `organization_guid`<br>`space_guid` |
+| `PUT /v2/service_instances/:instance_id/service_bindings/:binding_id` | `organization_guid`<br>`space_guid` |
```

Comment:
Let's add the appropriate stuff for Kube too, which would be the "namespace" just like all of the other calls.

Reply:
Done!